SimpleServer/log_collector.h

```diff
@@ -60,32 +61,70 @@ class LogCollectorHTTPServer {
   LogCollectorHTTPServer(int http_port,
                          std::ostream& ostream,
                          const std::string& route = "/log",
-                         const std::string& response_text = "OK\n")
-      : http_port_(http_port), ostream_(ostream), route_(route), response_text_(response_text) {
-    HTTP(http_port_).Register(route_, [this](Request r) {
-      LogEntry entry;
-      entry.t = static_cast<uint64_t>(bricks::time::Now());
-      entry.m = r.method;
-      entry.u = r.url.url_without_parameters;
-      entry.q = r.url.AllQueryParameters();
-      entry.b = r.body;
-      entry.f = r.url.fragment;
-      // TODO(dkorolev): HTTP headers come here.
-      // entry.h = { ... }
-      ostream_ << JSON(entry, "log_entry") << std::endl;
-      r(response_text_);
-    });
+                         const std::string& response_text = "OK\n",
+                         const size_t tick_interval_ms = 1000)  // `0` to disable tick events.
```

Comment:
I'd go as far here as saying that `1000` should not be the default -- let's force the users to always set this value. It's performance-critical (RAM- and disk-critical) in certain occasions, so no surprises is a good thing.

And we have `bricks::time::MILLISECONDS_INTERVAL` to make it look substantially scary :-) Thanks!